plugins/woocommerce/src/Admin/RemoteInboxNotifications/Transformers/ArrayValues.php

```diff
@@ -21,6 +21,10 @@ class ArrayValues implements TransformerInterface {
 	 * @return mixed
 	 */
 	public function transform( $value, stdClass $arguments = null, $default = null ) {
+		if ( ! is_array( $value ) ) {
+			return $default;
```

Comment:
Same as above. What do you think of setting the default value to an empty array? I think it makes more sense as the caller expects the return value to be an array.

Reply:
Changed in 1eb562755ff06c1e9e6fe8f855ab71001307bf0b. I also updated default value for `ArrayFlatten`.